.github/workflows/test-eclhelper.yml

```diff
@@ -0,0 +1,19 @@
+name: Check that eclhelper interface has not changed
+
+on:
+  pull_request:
+    branches:
+      - "master"
+      - "candidate-*"
+
+jobs:
+  build:
+    name: "Check eclhelper interface"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+      - name: Check eclhelper
+        run: git diff HEAD^ --exit-code -- rtl/include/eclhelper.hpp
```

Comment:
if there's a difference, doesn't it then need to see if ACTIVITY_INTERFACE_VERSION has been increased, and only fail if it hasn't ? Or if eclhelper interfaces are extended, how does the PR pass ?

Reply:
If anythinfg has changed in eclhelper.hpp has changed, it fails. That is deliberate. It can then be merged only by invoking the administrator override, which therefore raises a pretty big red flag (literally) to make sure the change is looked at very carefully.

I would want to look very carefully at any PR that DID change the interface verison too (tom make sure that it needed to, for example - we don't want spurious changes to the version either, though they are preferable to missing changes that WERE needed)

Reply:
ok, I wondered if that was the intention.